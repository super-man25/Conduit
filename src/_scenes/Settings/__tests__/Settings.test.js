import React from 'react';
import { shallow } from 'enzyme';
import { Settings } from '../index';

describe('<Settings />', () => {
  const props = {
    authState: { model: { id: 1, firstName: 'groot', lastName: 'groot' } },
    authActions: {}
  };

  it('renders correctly', () => {
    const wrapper = shallow(<Settings {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
