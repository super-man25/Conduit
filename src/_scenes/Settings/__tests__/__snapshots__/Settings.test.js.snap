// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Settings /> renders correctly 1`] = `
<styled.div>
  <Connect(SiteHeaderPresenter)
    auth={
      Object {
        "firstName": "groot",
        "id": 1,
        "lastName": "groot",
      }
    }
    authActions={Object {}}
  />
  <styled.div>
    <styled.div>
      <styled.div
        padding="2rem"
      >
        <Flex
          align-items="center"
          direction="column"
        >
          <styled.div
            key="/settings/team"
            padding="2rem 0"
          >
            <Styled(NavLink)
              activeStyle={
                Object {
                  "color": "#024A8F",
                  "textShadow": "0 0 0.5px #024A8F",
                }
              }
              to="/settings/team"
              weight="light"
            >
              Team Settings
            </Styled(NavLink)>
          </styled.div>
          <styled.div
            key="/settings/user"
            padding="2rem 0"
          >
            <Styled(NavLink)
              activeStyle={
                Object {
                  "color": "#024A8F",
                  "textShadow": "0 0 0.5px #024A8F",
                }
              }
              to="/settings/user"
              weight="light"
            >
              User Settings
            </Styled(NavLink)>
          </styled.div>
          <styled.div
            key="/settings/users"
            padding="2rem 0"
          >
            <Styled(NavLink)
              activeStyle={
                Object {
                  "color": "#024A8F",
                  "textShadow": "0 0 0.5px #024A8F",
                }
              }
              to="/settings/users"
              weight="light"
            >
              View Users
            </Styled(NavLink)>
          </styled.div>
        </Flex>
      </styled.div>
    </styled.div>
    <styled.div>
      <Route
        component={[Function]}
        key="0"
        path="/settings/team"
      />
      <Route
        component={[Function]}
        key="1"
        path="/settings/user"
      />
      <Route
        component={[Function]}
        key="2"
        path="/settings/users"
      />
      <SecuredRoute
        component={[Function]}
        key="3"
        path="/settings/create-user"
      />
      <SecuredRoute
        component={[Function]}
        key="4"
        path="/settings/demo"
      />
    </styled.div>
  </styled.div>
</styled.div>
`;
